1-js/04-object-basics/01-object/article.md

```diff
@@ -436,48 +436,48 @@ for (let code in codes) {
 */!*
 ```
 
-The object may be used to suggest a list of options to the user. If we're making a site mainly for German audience then we probably want `49` to be the first.
+El objeto puede usarse para sugerir una lista de opciones al usuario. Si estamos haciendo un sitio principalmente para el público alemán, entonces probablemente queremos que `49` sea el primero.
 
-But if we run the code, we see a totally different picture:
+Pero si ejecutamos el código, veremos una imagen totalmente diferente:
 
-- USA (1) goes first
-- then Switzerland (41) and so on.
+- USA (1) va primero
+- Luego Switzerland (41) y así sucecivamente.
 
-The phone codes go in the ascending sorted order, because they are integers. So we see `1, 41, 44, 49`.
+Los códigos telefónicos van en orden ascendente porque son enteros. Entonces vemos  `1, 41, 44, 49`.
 
-````smart header="Integer properties? What's that?"
-The "integer property" term here means a string that can be converted to-and-from an integer without a change.
+````smart header="¿Propiedades de enteros? ¿Qué es eso?"
+El término "propiedad de enteros" aquí significa que una cadena se puede convertir a y desde desde un entero sin nigún cambio.
 
-So, "49" is an integer property name, because when it's transformed to an integer number and back, it's still the same. But "+49" and "1.2" are not:
+Entonces, "49" es un nombre de propiedad entero, porque cuando este se transforma a un entero y viceversa continua siendo el mismo. Pero "+49" y "1.2" no lo son:
 
 ```js run
-// Math.trunc is a built-in function that removes the decimal part
-alert( String(Math.trunc(Number("49"))) ); // "49", same, integer property
-alert( String(Math.trunc(Number("+49"))) ); // "49", not same "+49" ⇒ not integer property
-alert( String(Math.trunc(Number("1.2"))) ); // "1", not same "1.2" ⇒ not integer property
+// Math.trunc es una función incorporada que elimina la parte decimal
```

Comment:
`built-in` se refiere a las funciones que vienen por defecto dentro de JavaScript
```suggestion
// Math.trunc es una función nativa que elimina la parte decimal
```